README.md

```diff
@@ -1 +1,219 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 Portfolio Project <a name="about-project"></a>
+
+> This is a personal portfolio project created using HTML,CSS and, JavaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. HTML
+2. CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSShttps://developer.mozilla.org/en-US/docs/Web/JavaScript/">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added Index.html
+- Added style.css
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A Web browser.
+2. A Code editor
+3. A Version cotrol sytem(Gt-msc)
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run these commands:
+
+<!--
+
+```sh
+  cd my-folder
+  git clone git@github.com:Katended/hello_world.git
+```
+--->
+
+
+### Usage
+
+To run the project, do the following:
+
+Open index.html using a live server extension.
+
+### Run tests
+
+To run tests, run the following command:
+
+> Coming soon!
+
+### Deployment
+
+You can deploy this project using:
+
+> Coming soon!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 David Katende
+
+Connect with David:
+- GitHub: [@githubhandle](https://github.com/Katended)
+- LinkedIn: [LinkedIn](linkedin.com/in/katende-david-01627461)
+
+ 📫 How to reach me katendedavidjoga@gmail.com
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add header
+- [ ] Add footer
+- [ ] Add more styling
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
Kindly add the link to the issues page here, so people can directly go the issues page from here.